docs/post-upgrade/post-upgrade-deprecated-sn-connector.asciidoc

```diff
@@ -0,0 +1,62 @@
+[[post-upgrade-deprecated-sn-connector]]
+== Update a deprecated ServiceNow connector
+
+{sn} ITSM and SecOps connectors created in {stack} version <= 7.15 or earlier are marked as deprecated after you upgrade to version >= 7.16. Deprecated connectors have a yellow icon after their name and display a warning message when selected.
```

Comment:
The symbols `<=` render as a left-pointing arrow in published docs. Not sure what the preferred fix is: there might be a way to escape the characters, or maybe just change to `=<`?
![image](https://user-images.githubusercontent.com/87339667/142243457-4d467fdd-2b80-44f5-8a69-ce9340354e33.png)